db/migrate/20171016194116_create_inventories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateInventories < ActiveRecord::Migration[5.1]
+  def change
+    create_table :inventories do |t|
+      t.string :title
```

Comment:
you might want to have "not null" on these db columns, otherwise you can get empty data inserted into your titles and created_bys